src/db.rs

```diff
@@ -176,9 +202,64 @@ fn get_cutoff_timestamp(days: u64) -> Option<i64> {
         .expect("SystemTime should be after unix epoch");
     let valid_duration = Duration::from_secs(60 * 60 * 24 * days);
 
-    let cutoff_timestamp = current_timestamp
-    .checked_sub(valid_duration)?;
+    let cutoff_timestamp = current_timestamp.checked_sub(valid_duration)?;
     use std::convert::TryInto;
-    Some(cutoff_timestamp.as_millis().try_into().expect("We should be fewer than 2^63 milliseconds since start of unix epoch"))
+    Some(
+        cutoff_timestamp
+            .as_millis()
+            .try_into()
+            .expect("We should be fewer than 2^63 milliseconds since start of unix epoch"),
+    )
 }
 
+// Add values to the collection `holoports_status`
+pub async fn add_holoport_status(hs: HostStats, db: &Client) -> Result<(), ApiError> {
+    let hp_status: Collection<Document> = db
+        .database("host_statistics")
+        .collection("holoports_status");
```

Comment:
I think we have decided to change the name of the collection from `holoports_status` to `holoport_status`, which should be used across entire code.